dashboard/src/actions/comparisonActions.js

```diff
@@ -54,14 +54,56 @@ export const getQuisbyData = (dataset) => async (dispatch, getState) => {
   }
   dispatch({ type: TYPES.COMPLETED });
 };
-const COLORS = ["#8BC1F7", "#0066CC", "#519DE9", "#004B95", "#002F5D"];
-export const parseChartData = () => (dispatch, getState) => {
+const COLORS = [
+  CONSTANTS.COLOR1,
+  CONSTANTS.COLOR2,
+  CONSTANTS.COLOR3,
+  CONSTANTS.COLOR4,
+  CONSTANTS.COLOR5,
+];
+
+const getChartValues = (run, benchmarkType) => {
+  const benchmark = benchmarkType?.toLowerCase();
+  const chartTitle = {
+    uperf: `Uperf: ${uperfChartTitleMap[run.metrics_unit.toLowerCase()]} | ${
+      run.test_name
+    }`,
+    fio: `Fio: ${run.test_name} | ${run.metrics_unit.toLowerCase()}`,
+  };
+  const yaxisScale = {
+    uperf: run.metrics_unit,
+    fio: "Mb/sec",
+  };
```

Comment:
It would be nicer if this were named `yaxisTitle` instead of `yaxisScale`:  yes, it gets inserted into the `scales` option, but the value is used for the `title.text` field (and a "scale" _should_ be a numeric value...).

Reply:
Done